tests/Browser/LoginTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Tests\Browser;
+
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+
+class LoginTest extends DuskTestCase
+{
+    /**
+     * A Dusk test example.
+     *
+     * @return void
+     */
+    public function testExample()
+    {
+        $this->browse(function (Browser $browser) {
+            $browser->visit('/login')
+                    ->type('email', 'swarup@ranium.com')
+                    ->type('password', '12345678')
+                    ->press('LOGIN')
+                    ->assertPathIs('/task');
+        });
+    }
+}
```

Comment:
More tests needed. for ex, to check if a user is already logged in it should be redirected to the dashboard (or /task page here)

Reply:
ok sir

Reply:
You need to login for each test case, so can't check if the user is already logged in or not.

Reply:
There needs to be a test for negative scenario i.e. when you provide invalid login credentials. Assert whether you see an error message in that case and that the user is not logged in.

Reply:
ok sir